.changeset/four-plums-report.md

```diff
@@ -0,0 +1,10 @@
+---
+'hasura-auth': minor
+---
+
+Store refresh tokens in SHA256
+
+The existing refresh tokens remains stored in plain text to avoid breaking the ongoing connections
+Hashed refresh tokens are stored in a separate `auth.hashed_refresh_tokens` table. Every new token will be automatically stored in this table.
+
+The use of the `auth.refresh_tokens` table is deprecated, and the table will be removed in the future.
```

Comment:
Should we be more explicit about timeline and future major release here?

Like:

The table `auth.refresh_tokens` is now deprecated and will be removed at [date] in a new major release.

Reply:
I'm not sure. I would avoid hardcoding a release date, but I would rather put a tentative date in the release notes.
I also think think to set a date is related to our processes rather than this pull request: we can have a discussion about clarifying our release and deprecation policies in parallel to this work